### PR TITLE
Exclude test logs from SDL validation

### PR DESCRIPTION
**Pull request by wtgodbe**
Resolves (part of) https://github.com/aspnet/AspNetCore/issues/16835

I'm trying to exclude all files under "artifacts/Linux_Test_Results/" and friends.

I have an internal build running with this change: https://dnceng.visualstudio.com/internal/_build/results?buildId=420854

@sunandabalu PTAL

### CODE REVIEW

**Comment by sunandabalu** [comment]
The change itself looks fine to me, waiting to see the results of the test build.

**Comment by wtgodbe** [comment]
This decreased SDL time from 50 to 17 minutes 🍾

**Comment by wtgodbe** [comment]
@Pilchie @JunTaoLuo @dougbu now I'm thinking I should just backport this to https://github.com/aspnet/AspNetCore/pull/16911 instead of disabling in 3.1 - any objections?

**Comment by sunandabalu** [comment]
Policheck is failing with invalid path to /UEPATH 
```
Invalid value for /UEPATH: eng/PoliCheckExclusions.xml
```
https://dnceng.visualstudio.com/internal/_build/results?buildId=420854&view=logs&j=7d9eef18-6720-5c1f-4d30-89d7b76728e9&t=f511b583-5060-5810-7549-865816347c8e&l=287

**Comment by wtgodbe** [comment]
> Policheck is failing with invalid path to /UEPATH

Well never mind then. I'll try passing the full path

**Comment by wtgodbe** [comment]
New build: https://dnceng.visualstudio.com/internal/_build/results?buildId=421143

File: eng/PoliCheckExclusions.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<PoliCheckExclusions>
+	<Exclusion Type="FolderPathFull">LINUX_TEST_RESULTS|MACOS_TEST_RESULTS|WINDOWS_TEST_RESULTS</Exclusion>
```
Comment (dougbu) [inline eng/PoliCheckExclusions.xml]: If there's a doc describing the syntax of these exclusions, please add a comment linking to it

**Review by dougbu — APPROVED** [review]
Looks good but isn't SDK enabled in 'release/3.1' as well?

**Comment by wtgodbe** [comment]
> Looks good but isn't SDK enabled in 'release/3.1' as well?

It is, but for the sake of minimizing risk in GA I think disabling it there is best. This PR already had 2 iterations. We probably won't trigger anything new in 3.1 with the GA builds, and disabling SDL will speed up the internal builds by up to 50 minutes.

File: eng/PoliCheckExclusions.xml (hunk 1 above)
Comment (wtgodbe) [reply eng/PoliCheckExclusions.xml]: The doc is an internal microsoft sharepoint document, I don't know if I can/should check in a reference to it in the repo

Comment (dougbu) [reply eng/PoliCheckExclusions.xml]: Oh well

**Review by JunTaoLuo — APPROVED** [review]

**Comment by sunandabalu** [comment]
> New build: https://dnceng.visualstudio.com/internal/_build/results?buildId=421143

I see Policheck issues only for files under src, @wtgodbe can you confirm the right folders were excluded?

**Comment by wtgodbe** [comment]
Closing in favor of https://github.com/aspnet/AspNetCore/pull/17107

**Pull request closed without merging**